test/test_uninstall.sh

```diff
@@ -23,8 +23,7 @@ v=0.9.1
 (
   tfenv install ${v} || exit 1
   tfenv uninstall ${v} || exit 1
-  check_version ${v} && exit 1 || exit 0
-) || error_and_proceed "Uninstall of version ${v} failed"
+) || error_and_proceed "Uninstalling of version ${v} failed"
```

Comment:
Why was is removed? I think it still needs for the test of `tfenv uninstall`.

Reply:
Oups ! Sorry ... big mistake !